config/application.rb

```diff
@@ -72,7 +72,7 @@ class Application < Rails::Application
 
     # config.eager_load_paths accepts an array of paths from which Rails will eager load on boot if cache classes is enabled.
     # Defaults to every folder in the app directory of the application.
-    config.eager_load_paths = []
+    # config.eager_load_paths = []
```

Comment:
We can add more eager load paths to get more out of forked workers.  Also, we can remove the "require => false" for many of the gems or manually require them in a `pre_fork` method.